classes/fullstack/project1/server/src/config.ts

```diff
@@ -0,0 +1,8 @@
+import { config } from 'dotenv'
```

Comment:
remove this file and modify config/index.ts

Reply:
Done